src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -223,7 +223,6 @@ This token is generated automatically by the client software, but in some cases,
 openssl rand -hex 32 > jwtsecret
 ```
 
-
 #### Running an execution client {#running-an-execution-client}
```

Comment:
```suggestion
#### Running an execution client 
```

Reply:
Thank you for your review @vj-codes!
Since this one is an h4 heading, it can be converted by the markdown parser (e.g., https://ethereum.org/en/developers/docs/nodes-and-clients/run-a-node/#running-an-execution-client). 

Are you sure we need to change also this one?